server/routes/termdb.singlecellSamples.ts

```diff
@@ -184,6 +183,8 @@ function validateDataNative(D: SingleCellDataNative, ds: any) {
 }
 
 function validateGeneExpressionNative(G: SingleCellGeneExpressionNative) {
+	G.singleCellGeneExpression2bins = {} // cache for binning gene expression values
```

Comment:
the cache should be {sample:{gene:binObj}} please document such and fix the logic

Reply:
yeah I missed that thanks